cypress/support/step-definitions/pages-steps.js

```diff
@@ -13,6 +13,7 @@ Then('My {word} Page is not visible', () => {
 });
 
 When('I go to {word} page', (word) => {
+  cy.wait(500);
```

Comment:
Can you explain why we have wait here?

Reply:
This is likely to wait for page animation to finish. Ihor can confirm cypress side of things.